### DISCUSSION TITLE
CloudBase 云开发部署遇到INVOKE_FUNCTION_FAILED错误

### DISCUSSION THREAD

**Discussion by elkan1788** [walinejs/waline#1478] [Q&A] [answered]
直接使得官方文档提供的 Deploy 按钮发布到腾讯云服务，部署过程一切正常，但在访问部署成功的页面时出现异常，信息参考如下：

![image1](https://user-images.githubusercontent.com/1217096/194292851-7439d917-fa4e-4ab8-be43-b262e2c4fdcf.png)

![image2](https://user-images.githubusercontent.com/1217096/194292923-a23fe4e2-5a68-4eed-a6d4-9e09f98350b6.png)

不确定是否还有遗漏其他参数配置呢？

> 上述这些参数在 `Vercel` 上面部署是成功的。

**Comment by lizheming** [comment, marked as answer]
same issue https://github.com/walinejs/waline/issues/1174

**Reply by elkan1788** [reply]
好的，那估计就只能是漫长的等待啦，谢谢。
